docs/openapi.yml

```diff
@@ -401,6 +405,15 @@ paths:
 
 components:
   parameters:
+    at:
+      name: at
+      in: query
+      description: The time the event happened at
```

Comment:
_Minor_: Given that the `at` query param is used to set the transition time of a given run, what are your thoughts on the description below?

```
An ISO-8601 timestamp representing the time when the run transitioned.
```